packages/swap/src/fixtures/ErrorBoundary.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {Component, ReactNode} from 'react'
+import {View, Text} from 'react-native'
+
+interface Props {
+  children: ReactNode
+}
+
+interface State {
+  hasError: boolean
+  error?: Error
+}
+
+export class ErrorBoundary extends Component<Props, State> {
+  state: State = {
+    hasError: false,
+  }
+
+  static getDerivedStateFromError(error: Error): State {
+    return {hasError: true, error}
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return (
+        <View testID="hasError">
+          <Text>hasError</Text>
```

Comment:
hey @MichalSzorad yes they are fixtures for the tests, so in the tests I just get the `testID` `hasError` basically everything else is irrelevant, are you suggesting to remove the `<Text>` component ? this could be done, the only reason to be here, is when the test fails Jest spill it on console, so its simpler to understand